docs/datamodel/computeds.rst

```diff
@@ -53,9 +53,9 @@ field is referenced in a query.
 
 .. warning::
 
-  Volatile functions are not allowed in computed properties defined in schema.
-  This means that, for example, your schema-defined computed property cannot
-  call :eql:func:`datetime_current`, but it *can* call
+  Volatile and modifying functions are not allowed in computed properties
+  defined in schema. This means that, for example, your schema-defined computed
+  property cannot call :eql:func:`datetime_current`, but it *can* call
```

Comment:
Maybe we can add a link to the corresponding docs for modyfing functions so that it's clear what we refer to?